cabal-install/bootstrap.sh

```diff
@@ -261,6 +261,15 @@ HACKAGE_SECURITY_VER="0.5.3.0"; HACKAGE_SECURITY_VER_REGEXP="0\.5\.((2\.[2-9]|[3
                        # >= 0.5.2 && < 0.6
 TAR_VER="0.5.1.0";     TAR_VER_REGEXP="0\.5\.([1-9]|1[0-9]|0\.[3-9]|0\.1[0-9])\.?"
                        # >= 0.5.0.3  && < 0.6
+# These two are temporary: https://github.com/jgm/zip-archive/pull/47
```

Comment:
Not actually a blocker for merging, right? I'd be OK with merging as-is and then fixing up when the PR is merged upstream.

Reply:
Nope, not a blocker. That's to remind us when it does get merged (hopefully, I don't see a reason it wouldn't be).